src/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import PropTypes from 'prop-types';
+import GithubSvg from './about-us/githubSvg';
+
+export default function Footer(props){
+  const enigmaBay = <a href='https://github.com/EnigmaBay' className='themed-hyperlink' data-theme={props.dataTheme}>EnigmaBay</a>;
+  const mit = <a href='https://opensource.org/licenses/MIT' className='themed-hyperlink' data-theme={props.dataTheme}>MIT License</a>;
```

Comment:
Can we point this to our actual license file in GH instead?

Reply:
Absolutely.